xarray/core/ops.py

```diff
@@ -28,6 +28,11 @@
     bn = np
     has_bottleneck = False
 
+try:
+    import dask.array as da
```

Comment:
I moved he dask array related code to `duck_array_ops`. That's probably a more natural place for this.

Actually: maybe we should have another module  for dask array operations, and then put the duck type functions in `dask_array_ops`.